examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
```

Comment:
On Thu, Jun 04, 2015 at 02:23:16PM -0700, Jeromy Johnson wrote:
> oh yeah... i just picked some random hashes. I can make them real,
> but then we have to worry about pinning them forever.

Can we use part of the current ipfs.io site?  For example [1]?  Using
the non-root site avoids self-referential trouble with this example
file being under the root, but using something from ipfs.io means we
get an interesting, complicated page that we're already pinning (we
just need to bump the hashes here when we drop old ipfs.io pins).

http://gateway.ipfs.io/ipfs/QmeYYwD4y4DgVVdAzhT7wW5vrvmbKPQj8wcV2pAzjbj886/